packages/transformer/src/IModelTransformer.ts

```diff
@@ -457,26 +475,42 @@ export class IModelTransformer extends IModelExportHandler {
   }
 
   /** Iterate all matching ExternalSourceAspects in the provenance iModel (target unless reverse sync) and call a function for each one. */
-  private forEachTrackedElement(fn: (sourceElementId: Id64String, targetElementId: Id64String) => void): void {
-    if (!this.provenanceDb.containsClass(ExternalSourceAspect.classFullName)) {
+  public static forEachTrackedElement(args: {
+    provenanceSource: IModelDb;
+    provenanceLinker: IModelDb;
```

Comment:
let's rename this to provenanceDb for consistency, I wasn't sure about this name anyway.